src/components/ScheduleTable/CustomCard/CustomCard.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles({
+  root: {
+    minWidth: 200,
+    minHeight: 86,
+  },
+  title: {
+    fontSize: 14,
+  },
+});
+
+const CustomCard = ({ subject = '', teacher = '', onClickHandler, startTime, endTime }) => {
+  const classes = useStyles();
+
+  return (
+    <Card
+      className={classes.root}
+      variant='outlined'
+      onClick={() => {
+        onClickHandler(startTime, endTime);
+      }}
```

Comment:
Поднять callback в компонент выше, тогда компонент чище будет
```suggestion
      onClick={onClickHandler}
```

Reply:
исправлен